components/sso-hostobject/org.wso2.carbon.hostobjects.sso/src/main/java/org/wso2/carbon/hostobjects/sso/internal/builder/AuthReqBuilder.java

```diff
@@ -53,90 +46,25 @@
      * @return AuthnRequest Object
      * @throws Exception error when bootstrapping
      */
-    public AuthnRequest buildAuthenticationRequest(String issuerId, String nameIdPolicy) throws Exception {
+    public AuthnRequest buildAuthenticationRequest(String issuerId, String acsUrl, boolean isPassive,
+            String nameIdPolicy) throws Exception {
```

Comment:
Throwing generic exception fixed in #1494 (https://github.com/wso2/carbon-apimgt/commit/a3c032ab9f7593c6e11cdf1e91fa04d5cc3de991)